projects/application/DatSphere-en.md

```diff
@@ -0,0 +1,62 @@
+# Project Name DatSphere
+
+## Category 
+application
+
+## Project Description
+An open-source, decentralized drive service built on ipfs, filecoin
+
+## Project Status
+under development
+### Done
+- Account Module  
+    - Login
+    - Register
+    - Password Change
+    - Reset Password
+- Drive Module
+    - navigate files
+    - upload file/folder
+    - create and update text-file/folder
+    - download file/folder
+- IPFS Module
+    - import remote ipfs hash
+    - generate ipfs hash for uploaded files
+### Todo
+- integrate FileCoin
+
+## Previews
+![Image](https://drive.google.com/file/d/1rICFF2y_pn3eH7_g-_P8WoSIjN3aqPfx/view)
+![Image](https://drive.google.com/file/d/1l4f4UqyxT-3eteWeZ2Dr3BMr7i8nQW7L/view)
+
+## Target Audience
+normal user, developer
+
+## Rough estimated user base (if applicable)
+
+
+## Github repo
+https://github.com/cgpiao/datsphere
+
+## Website
+https://datsphere.com
```

Comment:
I have created an account and tried the features. But I failed at:
+ upload file
+ create a text file. 
Could you please take a look at and give us more updates so that I can see how it is using IPFS or Filecoin ?

Reply:
the upload file and create text file function has been fixed. You may find the file on IPFS or upload/index a file by using IPFS CID. As for Filecoin, we are working on 'sync to chain' functions, will be able to get you back soon.